### DISCUSSION TITLE
NextJS: React does not recognize the `dataTw` prop

### DISCUSSION THREAD

**Discussion by prateekkarki** [ben-rogerson/twin.macro#174] [Q&A] [answered]
I am using twin.macro on my NextJS app with emotion. I keep getting this error on my console. It is because of the data-tw prop that is passed for debugging. 
![image](https://user-images.githubusercontent.com/8191488/97104453-0fe1bb00-16dc-11eb-91a0-2a3b9f6b2fcc.png)

Does this warning represent something wrong I might have done during setup? Is there an underlying issue because of this?
If not, is there a way I can suppress this warning.
I am using the same setup as explained in [Next/Emotion Docs](https://github.com/ben-rogerson/twin.macro/blob/master/docs/emotion/next.md).

**Comment by ben-rogerson** [comment, marked as answer]
That's interesting - I haven't seen that error before, perhaps it's a server rendering issue? 
[Twin adds the prop as kebab case](https://github.com/ben-rogerson/twin.macro/blob/d3e5c95620a132b55d039c0900810b2e380a3350/src/macro/debug.js#L13).

You can turn off the `data-tw` prop feature in your [twin config](https://github.com/ben-rogerson/twin.macro/blob/master/docs/emotion/next.md#options) with `debugProp`:

```js
debugProp: false
```

**Reply by prateekkarki** [reply]
Later I noticed that it was only getting the warning for CloudinaryContext component from cloudinary-react. I had used it as below to save an extra div 
```
<CloudinaryContext
   cloudName={process.env.NEXT_PUBLIC_CLOUDINARY_NAME}
    css={tw`rounded-t-md w-full h-64 overflow-hidden`}
>
    <Image .../>
```

I added css to a div and it fixed it. Thanks for your help 👍
